index.js

```diff
@@ -70,27 +70,33 @@ else {
     console.log('generating files...');
 }
 
+const ignore = ['**/node_modules/**/*'];
+try {
+    const puffIgnore = fs.readFileSync(path.join(process.cwd(), '.puffignore'), { encoding: 'utf8' })
+    lines = puffIgnore.match(/[^\r\n]+/g); 
+    ignore.push(...lines.map(x => {
+        const trimmed = x.trim();
+        if(trimmed.startsWith("#") || !trimmed) return undefined;
+        return trimmed;
+    }).filter(x => x));
+} catch(e) {}
```

Comment:
I don't like swallowing exceptions. can we emit a warning to the console?

Reply:
There's no unit test on the project so I don't think it would be part of the scope adding it now. 
I was using the try/catch as a condition but just found out that only fx.exists is deprecated, fx.existsSync is fine.